procedures/accounts_setup.md

```diff
@@ -0,0 +1,187 @@
+    release@m.c can contain security-sensitive information. Do not automatically forward your email to a system that is not under Mozilla's control. Same goes for releng@mozilla.com.
+
+### Calendar
+
+Like mail, we now use [Google calendar](https://www.google.com/calendar/)
+
+You'll want to subscribe to the following public calendars:
+* [RelEng MoCo calendar](https://calendar.google.com/calendar/u/0?cid=bW96aWxsYS5jb21fMmQzMjM0MzMzMzM1MzAzNjMxMmQzOTM3MzdAcmVzb3VyY2UuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
+* [RelEng PTO calendar](https://calendar.google.com/calendar/u/0?cid=Y190cWEzaHQ5bXB0ODN1djJzZDJsYjVsaTVva0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t)
+
+Talk to your manager/mentor to get added to the various other private calendars as appropriate.
+
+### Matrix
+
+The majority of day-to-day communication in releng happens on Matrix. More details on this can be found on [wiki Matrix](https://wiki.mozilla.org/Matrix).
+Please talk to your manager about the private/public channels you should be in.
+
+### Slack
+
+Some parts of Mozilla prefer Slack to Matrix, more info on [mana](https://mana.mozilla.org/wiki/display/SD/Slack).
+
+### Zoom
+
+For video communication we use Zoom. More details can be found on [mana](https://mana.mozilla.org/wiki/display/AVSE/Getting+Started+With+Zoom)
+
+### Wiki
+
+Historical pieces of Mozilla information can be found in wiki.
+TODO - ask Heitor for instructions on how to get wiki.m.o account
+
+### Docs
+
+Our single-source of docs is https://docs.mozilla-releng.net/en/latest/ which is sourced from https://github.com/mozilla/build-relengdocs.
+Don't be shy about making improvements to releng pages based on your experiences. Getting someone in releng to review your changes first is good practice. Just ask your team for more details.
+
+### Google Drive
+
+Google Drive (formerly Google docs) is a preferred way to share things these days. This includes spreadsheets and documents that will change a great deal over time.
+Google Drive access should be enabled with your email account when you start. If you need access to a particular document, talk to the document owner or your manager/mentor.
+
+## Bugzilla
+
+Almost everything at Mozilla goes through Bugzilla. [Create a Bugzilla account](https://bugzilla.mozilla.org/createaccount.cgi) if you have not already.
+You'll need a few tweaks to your account to get access to everything releng-related:
+* Add privileges for bugzilla group releng confidential (Can be done by manager or bugzilla admin)
+* Add your comms nickname & ldap username as "aliases" for your account
+* log into bugzilla & follow links "Preferences" -> "Account Information"
+* append the aliases, with a leading ':' and enclosed in brackets ('[]') to the "Real Name" field
+* e.g.: "John Doe [:jdoe]"
+
+The product to use is "Release Engineering." There are multiple possible components under that product, so take your best guess or ask for guidance from the team.
+TODO - to revisit once we reorg Bugzilla in November 2022.
```

Comment:
Note to self.

Reply:
I feel like Bugzilla should be mentioned first, as some of the above accounts require filing a bug in it.

Reply:
Good catch, moved at the top.